interface/wx/intl.h

```diff
@@ -66,6 +66,91 @@ struct wxLanguageInfo
 };
 
 
+/**
+    @class wxLocaleIdent
+    
+    Contains platform dependent language name.
+
+    Usage example:
+    @code
+      auto loc = wxLocaleIdent("fr").Region("BE").Modifer("euro");
+      assert( loc.GetName() == "fr_BE@euro" );
```

Comment:
Sorry, I know it's my own example, but it only works under Unix, so we should do something like
```suggestion
    #if defined(__WINDOWS__) || defined(__WXOSX__)
      wxASSERT( loc.GetName() == "fr_BE" );
    #elif defined(__UNIX__)
      wxASSERT( loc.GetName() == "fr_BE@euro" );
    #endif
```